Tasks/AzureResourceGroupDeployment/main.ts

```diff
@@ -15,7 +15,7 @@ function run(): Promise<void> {
                 return resourceGroupOperationsController.createOrUpdateResourceGroup();
             case "DeleteRG":
                 return resourceGroupOperationsController.deleteResourceGroup();
-            case "Select Resource Group":
+            case "Configure deployment options for vitual machine":
```

Comment:
You will need to handle back-compat. "Select Resource Group" value should still work for release definitions which were created earlier.